src/components/plots/MultiViewGrid.jsx

```diff
@@ -47,18 +145,16 @@ const MultiViewGrid = (props) => {
       }
 
       const plotsToAdd = _.difference(currentPlotUuids, previousPlotUuids);
-
       const newPlots = { ...plots };
 
-      plotsToAdd.forEach((plotUuid) => {
-        newPlots[plotUuid] = renderPlot(plotUuid);
+      plotsToAdd.forEach((uuid) => {
+        newPlots[uuid] = renderPlot(uuid);
       });
-
       setPlots(newPlots);
     }
   }, [multiViewConfig, plotConfigs]);
 
-  const spaceAlign = (multiViewConfig.plotUuids.length > 1)
+  const spaceAlign = (multiViewConfig?.plotUuids?.length > 1)
```

Comment:
Why are these conditionals necessary when they weren't before?

Reply:
I moved the generation of configs in this component so this can crash initially when there's no plots rendered